docs/index.rst

```diff
@@ -62,6 +64,12 @@ contacting support.
 +--------------------------------------------------+---------------+----------------+
 | :ref:`asyncpg`                                   | >= 0.22.0     | Yes            |
 +--------------------------------------------------+---------------+----------------+
+| :ref:`avro`                                      | \*            | Yes            |
++--------------------------------------------------+---------------+----------------+
+| :ref:`aws_lambda`                                | \*            | Yes            |
++--------------------------------------------------+---------------+----------------+
+| :ref:`azure_functions`                           | >= 1.0.0      | Yes            |
```

Comment:
@duncanhewett , should we have these versions in the docs for azure and aws?

Reply:
> @duncanhewett , should we have these versions in the docs for azure and aws?

@wantsui Are you asking if they should be on the [Python Compatibility Docs](https://docs.datadoghq.com/tracing/trace_collection/compatibility/python/) or somewhere else?

Reply:
@duncanhewett Asking if these should be published on the Python tracer API docs: https://ddtrace.readthedocs.io/en/stable/#supported-libraries

Reply:
I don't think there's a reason not to but I could be missing some context.

Reply:
I wasn't sure if there was any reason not to have this there already :)